### DISCUSSION TITLE
WebLogic Monitoring Exporter 2.1.1

### DISCUSSION THREAD

**Discussion by russgold** [oracle/weblogic-monitoring-exporter#253] [General]
### Additional Features
* String metrics now include the actual value as a qualifier by @russgold in https://github.com/oracle/weblogic-monitoring-exporter/pull/249
### Bug Fixes
* Metrics printing fixed on Windows by @russgold in https://github.com/oracle/weblogic-monitoring-exporter/pull/220
* WebLogic REST API sessions are now managed and should not grow by @russgold in https://github.com/oracle/weblogic-monitoring-exporter/pull/239
* Forbidden fields are now ignored by @russgold in https://github.com/oracle/weblogic-monitoring-exporter/pull/240
* Filtering always uses the "name" field by @russgold in https://github.com/oracle/weblogic-monitoring-exporter/pull/243

**Full Changelog**: https://github.com/oracle/weblogic-monitoring-exporter/compare/v2.1.0...v2.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/oracle/weblogic-monitoring-exporter/releases/tag/v2.1.1'>WebLogic Monitoring Exporter 2.1.1</a>.</em>

**Comment by mrsabapathy** [comment]
Hi, can we get metrics for JMS destinations (queue) such as Messages Current, Messaged Pending, Messages High, Messages Received etc?

**Reply by mrsabapathy** [reply]
Hi tagging for visibility
@russgold
